tests/contrib/hooks/test_ftp_hook.py

```diff
@@ -115,7 +115,9 @@ def test_retrieve_file(self):
         _buffer = six.StringIO('buffer')
         with fh.FTPHook() as ftp_hook:
             ftp_hook.retrieve_file(self.path, _buffer)
-        self.conn_mock.retrbinary.assert_called_once_with('RETR path', _buffer.write)
+        self.conn_mock.retrbinary.assert_called_once_with(
+            'RETR path',
+            _buffer.write)
```

Comment:
@OmerJog BTW, I don not think you should change this code to three lines, just keep it like before

```suggestion
            self.conn_mock.retrbinary.assert_called_once_with('RETR path', _buffer.write)
```

Reply:
@zhongjiajie I changed it as it gave flake8 errors "line too long"

Reply:
@OmerJog this repo flake8 length is 110, you can find in [this line](https://github.com/apache/airflow/blob/31318ba8eb5b136277fc1672813cb57e68989abf/.flake8#L2), and [source code](https://github.com/apache/airflow/blob/31318ba8eb5b136277fc1672813cb57e68989abf/tests/contrib/hooks/test_ftp_hook.py#L118) is only 86.